src/datapump/sync/sync.py

```diff
@@ -723,6 +723,125 @@ def get_latest_release(self) -> Tuple[str, List[str]]:
         return latest_release, source_uri
 
 
+class DISTAlertsSync(Sync):
+    """
+    Defines jobs to create new DIST alerts assets once a new release is available.
+    """
+
+    dataset_name = "umd__glad_dist_alerts"
+    source_bucket = "earthenginepartners-hansen"
+    source_prefix = "DIST-ALERT"
+    input_calc = """
+        np.where((A>=30) & (A<255) & (B>0) & (C>=2) & (C<255),
+            np.where(C<4, 20000 + B, 30000 + B),
+            -1
+        )
+    """
+
+    def __init__(self, sync_version: str):
+        super().__init__(sync_version)
```

Comment:
Replace this line by:

        self.sync_version = sync_version

I think your above line is left over when this was a subclass of DeforestationAlerts.  I'm not sure if your current line will even work, since __init__ is an abstract method in the superclass Sync.